content/en/platform/corda/4.8/open-source/key-concepts-time-windows.md

```diff
@@ -31,45 +31,40 @@ title: Time-windows
-However, if the party is not also compelled to commit to the associated transaction, it has a choice of whether or not
-to reveal this fact until some point in the future. As a result, we need to ensure that the notary either has to also
-sign the transaction within some time tolerance, or perform timestamping *and* notarisation at the same time. The
-latter is the chosen behaviour for this model.
+A party obtains a time-window signature in order to prove a transaction happened *before*, *on*, or *after* a particular point in time.
+As long as the party is not required to commit to the associated transaction, it can choose to reveal this fact at some point in the future. As a result the notary
+timestamps and notarizes at the same time.
 
-There will never be exact clock synchronisation between the party creating the transaction and the notary.
-This is not only due to issues of physics and network latency, but also because between inserting the command and
-getting the notary to sign there may be many other steps (e.g. sending the transaction to other parties involved in the
-trade, requesting human sign-off…). Thus the time at which the transaction is sent for notarisation may be quite
-different to the time at which the transaction was created.
+Thus, the time at which the transaction is sent for notarization may be
+different to the transaction creation time. There will never be exact clock synchronization between the party creating the transaction and the notary for two reasons:
+* Issues of physics and network latency
+* Between inserting the command and getting the notary to sign there may be many other steps, such as sending the transaction to other parties involved in the
+trade, or requesting human sign-off.
 
 ## Time-windows
 
-For this reason, times in transactions are specified as time *windows*, not absolute times. In a distributed system
-there can never be “true time”, only an approximation of it. Time windows can be open-ended (i.e. specify only one of
-“before” and “after”) or they can be fully bounded.
+Times in transactions are specified as time *windows*, not absolute times. Time-windows can be open-ended, and specify only
+“before” **or** “after”, or they can be fully bounded.
 
 {{< figure alt="time window" width=80% zoom="/en/images/time-window.gif" >}}
 
-In this way, we express the idea that the *true value* of the fact “the current time” is actually unknowable. Even when
-both a before and an after time are included, the transaction could have occurred at any point within that time-window.
+When both a before and an after time are included, the transaction could have occurred at any point within that time-window.
 
-By creating a range that can be either closed or open at one end, we allow all of the following situations to be
+By creating a range that can be either closed or open at one end, we allow all the following situations to be
 modelled:
```

Comment:
```suggestion
Time windows let you represent transactions that follow different models, such as those that occur:
```
Some more style guide notes here: we don't use "we" in docs. It's also ideal to rephrase wordy phrases such as "allows you to" with "lets" and to eliminate necessary phrases like "the following".

For lists, it's better to use a phrase that applies to all the bullet points in the intro section so it's not repeated.